docs/reference/repositories-metering-api/apis/repositories-meterings-body.asciidoc

```diff
@@ -95,23 +135,25 @@ grouped by request type.
 (long) Number of https://docs.microsoft.com/en-us/rest/api/storageservices/list-blobs[List Blobs] requests.
 `PutBlob`::
 (long) Number of https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob[Put Blob] requests.
-`PutBlobBlock`::
-(long) Number of  https://docs.microsoft.com/en-us/rest/api/storageservices/put-block[Put Block] requests.
+`PutBlock`::
+(long) Number of https://docs.microsoft.com/en-us/rest/api/storageservices/put-block[Put Block] and https://docs.microsoft.com/en-us/rest/api/storageservices/put-block-list[Put Block List] requests.
 
 Azure storage https://azure.microsoft.com/en-us/pricing/details/storage/blobs/[pricing].
 ======
 +
 .Properties of `request_counts` for repository type `GCP`
 [%collapsible%open]
 ======
-`storage.objects.get`::
-(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/get[get] requests.
-`storage.objects.list`::
-(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/list[list] requests.
-`storage.objects.resumable-insert`::
-(long) Number of https://cloud.google.com/storage/docs/performing-resumable-uploads[resumable insert] requests.
-`storage.objects.insert`::
-(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/insert[insert] requests.
+`GetObject`::
+(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/get[get object] requests.
+`ListObjects`::
+(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/list[list objects] requests.
+`ResumableInsertObject`::
+(long) Number of https://cloud.google.com/storage/docs/performing-resumable-uploads[resumable insert object] requests,
+even though resumable uploads can perform multiple http requests they are considered as a single operation for billing
+purposes.
+`InsertObject`::
+(long) Number of https://cloud.google.com/storage/docs/json_api/v1/objects/insert[insert object] requests.
```

Comment:
Does this include resumable uploads? I find it confusing how this relates to `ResumableInsertObject` given what you link to here provides docs on Simple, Multi-part, and Resumable uploads.
Do we even want to make the distinction between these three different upload types? Given that they are all billed the same way (with resumable uploads containing multiple HTTP requests), we could start by just exposing `InsertObject`, and if need be later, refine this into `InsertMultipartObject`, `InsertResumableObject`, etc. We can still track this internally at a more fine-grained level